packages/esm-active-visits-app/src/active-visits-widget/active-visits.component.tsx

```diff
@@ -175,10 +175,11 @@ const ActiveVisitsTable = () => {
                           <th colSpan={headers.length + 2}>
                             <ExtensionSlot
                               className={styles.visitSummaryContainer}
-                              extensionSlotName="visit-summary-slot"
+                              extensionSlotName="visit-detail-overview-slot"
                               state={{
-                                visitUuid: paginatedActiveVisits[index]?.visitUuid,
+                                visit: paginatedActiveVisits[index]?.visit,
                                 patientUuid: paginatedActiveVisits[index]?.patientUuid,
+                                listViewOverride: true,
                               }}
                             />
```

Comment:
Is this a single-use extension slot (i.e., only one extension ever goes in it, and it should always go in it)? Can we replace it with just using the component directly?

Reply:
I realize now, we made this into an extension slot as this gives orgs the flexibility to use their own widget to show this data. At AMPATH, we're using obs to recording medication orders as well as concepts that may be specific to us.

Reply:
Where does that stuff show up? Don't we expect that kind of customization to be needed for most implementations? Can we accomplish that by making the widget configurable?

Reply:
I feel like this would be comlicated to do via configuration as this is based on context. The visit type, program encrollment, the role of the user may all be implementation level factors that really require logic to sort out. I'd like to merge this for now as I believe it's blocking us from moving forward. If we change the approach later it won't be a big deal to refactor this